CONTRIBUTING.md

```diff
@@ -121,6 +121,8 @@ make install-test-requirements
 make install-pre-commit
 ```
 
+> **_NOTE:_** If Spark/PySpark/Hive tests for datasets are failing it might be due to the lack of Java>8 support from Spark. You can try using `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)` which works under MacOS or other workarounds. [Reference](https://stackoverflow.com/questions/53583199/pyspark-error-unsupported-class-file-major-version-55)
```

Comment:
I'd personally shorten this to:

```suggestion
> *Note*: If your Spark/PySpark/Hive tests for datasets are failing, make sure you have Java 8 installed. Spark doesn't currently support Java >8.
```

Reply:
Agreed this would solve things. However, I actually didn't need to install Java8 but only set that flag. Keeping maybe the link or the flag could prevent some unnecessary digging around and installing things for some?
Maybe take your suggestion and add: Mac users can try the suggestion here: [Reference]()

Reply:
Ah, gotcha.Can you change `> **_NOTE:_**` to `> *Note:*` to be consistent with the other notes in the section? :)

Reply:
Done.